src/Notes/NavigationFeedbackFollowUp.php

```diff
@@ -24,6 +24,13 @@ class NavigationFeedbackFollowUp {
 	 */
 	const NOTE_NAME = 'wc-admin-navigation-feedback-follow-up';
 
+	/**
+	 * Should this note exist? (The navigation feature should exist.)
+	 */
+	public static function is_applicable() {
+		return Features::exists( 'navigation' );
```

Comment:
If we're going by inverse logic of note creation, we'd want to check if the feature is enabled:

```suggestion
		return Features::is_enabled( 'navigation' );
```

But this note may be useful even if the feature is disabled.  Do we want to still keep this note even if the feature is disabled, @pmcpinto?  Title: "Share your thoughts on the new WooCommerce navigation"

I'm secretly hoping that all notes can just be inverse logic for creation or deletion so we can use `is_applicable` to clean up all note creation/deletion.

Reply:
Oh, this is a good point. My first thought was that this note won't be created unless `NavigationFeedback` is created, which only happens if navigation is enabled, but if they didn't like it and turned it back off we'd want to know that too.

Reply:
> But this note may be useful even if the feature is disabled. Do we want to still keep this note even if the feature is disabled, @pmcpinto? Title: "Share your thoughts on the new WooCommerce navigation"

Nice catch. It makes sense to keep the note. Thanks for the suggestion!

Reply:
So where we land is: keep checking `Feature::exists('navigation')` in `is_applicable()`, with apologies to @joshuatf for the asymmetry. Yes?

Reply:
Yup!  That sounds correct.